chess-AI/figshare_api.py

```diff
@@ -0,0 +1,195 @@
+"""Module for retrieving and uploading articles through the
```

Comment:
Please address all these pylint warnings https://pastebin.com/dLGf8SSn

Reply:
What pylint do you run? I ran pylint on that and got a 8.37/10 and the only errors were some string formatting which was just how they did it on code that I used from the figshare docs.